data/shell-completions/zsh/_meson

```diff
@@ -29,7 +29,7 @@
 local curcontext="$curcontext" state line
 local -i ret
 
-local __meson_backends="(ninja xcode ${(j. .)${:-vs{,2010,2015,2017}}})"
+local __meson_backends="(ninja "${(j. .)${:-vs{,2010,2015,2017,2019,2022}}}" xcode none)"
 local __meson_build_types="(plain debug debugoptimized minsize release)"
 local __meson_wrap_modes="(WrapMode.{default,nofallback,nodownload,forcefallback})"
```

Comment:
```suggestion
local __meson_wrap_modes="(default nofallback nodownload forcefallback nopromote)"
```

This change is unrelated to the other ones, but I think it's appropriate to mention it here since you're touching ZSH completion.

I have no idea what is the purpose of `WrapMode.{default,nofallback,nodownload,forcefallback}`.

```
meson setup --wrap-mode=WrapMode.default
```

results in

```
usage: meson setup [-h] [--prefix PREFIX] [--bindir BINDIR] [--datadir DATADIR] [--includedir INCLUDEDIR] [--infodir INFODIR] [--libdir LIBDIR] [--licensedir LICENSEDIR]
                   [--libexecdir LIBEXECDIR] [--localedir LOCALEDIR] [--localstatedir LOCALSTATEDIR] [--mandir MANDIR] [--sbindir SBINDIR]
                   [--sharedstatedir SHAREDSTATEDIR] [--sysconfdir SYSCONFDIR] [--auto-features {enabled,disabled,auto}]
                   [--backend {ninja,vs,vs2010,vs2012,vs2013,vs2015,vs2017,vs2019,vs2022,xcode,none}] [--genvslite {vs2022}]
                   [--buildtype {plain,debug,debugoptimized,release,minsize,custom}] [--debug] [--default-library {shared,static,both}] [--errorlogs]
                   [--install-umask INSTALL_UMASK] [--layout {mirror,flat}] [--optimization {plain,0,g,1,2,3,s}] [--prefer-static] [--stdsplit] [--strip]
                   [--unity {on,off,subprojects}] [--unity-size UNITY_SIZE] [--warnlevel {0,1,2,3,everything}] [--werror]
                   [--wrap-mode {default,nofallback,nodownload,forcefallback,nopromote}] [--force-fallback-for FORCE_FALLBACK_FOR] [--vsenv] [--pkgconfig.relocatable]
                   [--python.bytecompile PYTHON.BYTECOMPILE] [--python.install-env {auto,prefix,system,venv}] [--python.platlibdir PYTHON.PLATLIBDIR]
                   [--python.purelibdir PYTHON.PURELIBDIR] [--python.allow-limited-api] [--pkg-config-path PKG_CONFIG_PATH]
                   [--build.pkg-config-path BUILD.PKG_CONFIG_PATH] [--cmake-prefix-path CMAKE_PREFIX_PATH] [--build.cmake-prefix-path BUILD.CMAKE_PREFIX_PATH] [-D option]
                   [--native-file NATIVE_FILE] [--cross-file CROSS_FILE] [-v] [--fatal-meson-warnings] [--reconfigure] [--wipe] [--clearcache]
                   [builddir] [sourcedir]
meson setup: error: argument --wrap-mode: invalid choice: 'WrapMode.default' (choose from 'default', 'nofallback', 'nodownload', 'forcefallback', 'nopromote')
```

`WrapMode.` modes are clearly invalid.

Reply:
There's a lot of broken things like that I was going to fix in another PR later. This PR is mostly to add the missing subcommands. I did commit that fix though, thanks!